server/package.json

```diff
@@ -4,6 +4,8 @@
   "private": true,
   "scripts": {
     "start": "node ./bin/www",
+    "start:autoreload": "nodemon ./bin/www",
```

Comment:
Si on essaye de lancer la commande *start:autoreload* depuis le dossier server on a l'erreur :
nodemon: command not found

Il faut installer nodemon comme dependance dev (devDependencies)